doc/src/build/install.md

```diff
@@ -4,69 +4,67 @@ title: Install Sui to Build
-1. Request [SUI tokens](#sui-tokens) to evaluate Devnet and Sui Wallet
+1. Install [prerequisites](#prerequisites) for your operating system.
+1. Install the [Sui binaries](#install-sui-binaries).
+1. Configure an [Integrated Development Environment (IDE)](#integrated-development-environment).
+1. Request [SUI tokens](#sui-tokens) to evaluate Devnet and Sui Wallet.
 1. Optionally, download the [source code](#source-code) to have local
    access to examples and contribute to Sui.
 
-## Branches of the Sui repo
+## Sui repository
 
-The Sui repo includes two primary branches, `devnet` and `main`.
+The Sui repository includes two primary branches, `devnet` and `main`.
 
- * The `devnet` branch includes the latest stable build of Sui. Choose the `devnet` branch if you want to build or test on Sui. If you encounter an issue or find a bug, it may already be fixed in the `main` branch. To submit a Pull Request (PR), you should push commits to your fork of the `main` branch.
- * The `main` (Latest build) branch includes the most recent changes and updates. Use the `main` (Latest build) branch if you want to contribute to the Sui project. The `main` branch may include unreleased changes, or introduce changes that cause issues in apps created using an earlier version.
+ * The `devnet` branch includes the latest stable build of Sui. Choose the `devnet` branch if you want to build or test on Sui. If you encounter an issue or find a bug, it may already be fixed in the `main` branch. To submit a pull request (PR), you should push commits to your fork of the `main` branch.
+ * The `main` branch includes the most recent changes and updates. Use the `main` branch if you want to contribute to the Sui project. The `main` branch may include unreleased changes, or introduce changes that cause issues in apps created using an earlier version.
 
-## Choose the documentation branch
+## Documentation in the Sui repository
 
-This documentation is built from the same branches, `main` and `devnet`. The `main` branch includes the latest additions and updates to the documentation. You can view the content to learn about upcoming updates to the documentation, but the information may not be accurate or up-to-date for the features and functionality available in the `devnet` branch. In most cases, you should view the `devnet` version of the documentation.
+The `main` and `devnet` branches of the Sui repository contain the relevant documentation for each branch. A version toggle on the documentation site enables you to switch between `main` branch content (labeled **Latest build**) and `devnet` branch content (labeled **Devnet**). Make sure the toggle is set to **Devnet** to learn how to install, configure, and build on Sui. The content in **Latest build** is useful to learn about potential updates to Sui, but is not guaranteed to be accurate for the features and functionality available in the `devnet` branch.  
```

Comment:
we can't currently guarantee that the devnet docs are accurate and current with the devnet branch of the repo either.